### PR TITLE
Added Support for variable array types

### PR DESCRIPTION
**Pull request by LukasParke**
@Roger13 This is what a solution to #8 Looks like in my mind, but its worth noting that this doesn't address the original issue submitters use case of variable example entries.

I'm not sure they best way to tackle that with your current recursive implemention.

### CODE REVIEW

**Comment by LukasParke** [comment]
Also apologies on prettier having a field day on the whitespace

**Comment by LukasParke** [comment]
I take it back, I added the examples successfully, but the indentation isnt lining up

File: swaggerGen.js [hunk 1]
```diff
@@ -15,183 +15,223 @@ function convert() {
...
+    // ---- Begin example scope ----
+    if (document.getElementById("requestExamples").checked) {
+      console.log(examples);
+      outSwagger +=
+        "," +
+        indentator +
+        '"example": ' +
+        JSON.stringify(Array.from(examples), null, "\t");
+    }
+  }
```
Comment (Roger13) [inline swaggerGen.js]:
```suggestion
    // ---- Begin example scope ----
    if (document.getElementById("requestExamples").checked) {
      outSwagger += ","
      outSwagger += indentator + '"example": ' + JSON.stringify(
        Array.from(examples), null, '\t'
      ).replaceAll('\n', indentator)
    }
    // ---- End example scope ----
```

**Comment by Roger13** [comment]
Fixed the indentation issue.
Thanks a lot for tackling this! 
🙌

**Pull request merged**